src/components/app/CurrentProfileUploadedInformationLoader.js

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Thought: this class might be a good candidate to try out React hooks. I didn't want to change it now though, because I want it LANDED :-)